pkg/scheduler/scheduler.go

```diff
@@ -516,10 +516,13 @@ func (sched *Scheduler) scheduleOne(ctx context.Context) {
 
```

Comment:
L513: we should probably make `metrics.PodScheduleError` calls be consistently ordered too while we're at it? It's metrics -> recordSchedulingFailure everywhere else except L513.

Reply:
SGTM

Reply:
done